src/core/main.cpp

```diff
@@ -528,6 +528,57 @@ int main( int argc, char * * argv )
 		}
 	}
 
+	// Test file argument before continuing
+	QFileInfo fileInfoLoad( fileToLoad );
+	QFileInfo fileInfoImport( fileToImport );
+	if( !fileToLoad.isEmpty() )
```

Comment:
> Shouldn't we only instantiate fileInfoLoad if the QString isn't null or empty? This would also remove the need for setting an empty QString above.

No, I don't think so, but this part is essentially the same as before. I just moved it and expanded on the functionality.


> you have a commented line that should be removed

comment removed